src/components/layout/MainLayout/MainLayout.js

```diff
@@ -7,19 +7,27 @@ import Brands from '../../features/Brands/BrandsContainer';
 import StickyBarContainer from '../StickyBar/StickyBarContainer';
 import LatestBlog from '../../features/LatestBlog/LatestBlogContainer';
 
-const MainLayout = ({ children }) => (
-  <div>
-    <Header />
-    <StickyBarContainer />
-    {children}
-    <LatestBlog />
-    <Brands />
-    <Footer />
-  </div>
-);
+const MainLayout = ({ children, changeMode }) => {
+  React.useEffect(() => {
+    window.addEventListener('resize', changeMode);
+  });
+  window.removeEventListener('resize', changeMode);
```

Comment:
To źle jest zapisane, poszukaj jak zrobić `componentWillUnmount` w `useEffect` ;)

Reply:
`useEffect` też jest zły, powinnien odpowiadać `componentDidMount` ;)

Reply:
czy teraz jest ok? @kamilwilk95 reszta poprawiona

Reply:
nie do końca, `useEffect` teraz się wykonuje za każdym rerenderem a powinien zastąpić `componentDidMount`

Reply:
const MainLayout = ({ children, changeMode }) => {
  useEffect(() => {
    window.addEventListener('resize', changeMode);

    return () => {
      window.removeEventListener('resize', changeMode);
    };
  }, [changeMode]);

jakoś tak? obawiam się, że utknęłam, a mam jeszcze inne taski, więc przydałaby mi się pomoc :( 
@kamilwilk95 
@maciejburak może Ty się w tym orientujesz?